chainio/clients/wallet/fireblocks_wallet.go

```diff
@@ -97,9 +97,9 @@ func (t *fireblocksWallet) getAccount(ctx context.Context) (*fireblocks.VaultAcc
 		if err != nil {
 			return nil, fmt.Errorf("error listing vault accounts: %w", err)
 		}
-		for _, a := range accounts {
+		for i_a, a := range accounts {
```

Comment:
why `i_a` ? - I think this can just be `i` @pablodeymo

Reply:
Fixed. Thanks!